OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -162,13 +162,13 @@ public override bool Tick(Actor self)
 
 			if (mobile.Info.CanMoveBackward && self.World.WorldTick - startTicks < mobile.Info.BackwardDuration && Math.Abs(firstFacing.Angle - mobile.Facing.Angle) > 256)
 			{
-				actorFacingModifier = new WAngle(512);
-				firstFacing += actorFacingModifier;
+				ActorFacingModifier = new WAngle(512);
+				firstFacing += ActorFacingModifier;
 			}
 			else
-				actorFacingModifier = WAngle.Zero;
+				ActorFacingModifier = WAngle.Zero;
 
-			if (firstFacing != mobile.Facing)
+			if (!mobile.Info.TurnsWhileMoving && firstFacing != mobile.Facing)
```

Comment:
I think it makes sense. If you don't need to turn you don't turn. The purpose of the move activity it to move the target, and turning is merely a means to an end.

Reply:
In the case of docking, it would look weird for an actor to first turn to the movement facing and then turn to the docking facing. Same with queued up deploy.

Reply:
One could argue that an actor that can turn while moving should slide into position during docking anyway, so this sounds more like an issue with the target facing for the docking activity. In the end I don't really care either way until we have a practical usecase.

Reply:
when thinking of TD or TS refineries you really need to slide at the correct angle. Otherwise it will both look weird and don't matchup with the refinery model